server_manager/www/views/server_view/access_key_data_table/index.ts

```diff
@@ -0,0 +1,110 @@
+/**
+ * Copyright 2025 The Outline Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {LitElement, html} from 'lit';
+import {customElement, property} from 'lit/decorators.js';
+
+import './data_table';
+import {NUMERIC_COMPARATOR} from './data_table';
+import type {DataTableColumnProperties} from './data_table';
+
+import './access_key_status';
+import './access_key_controls';
+import './access_key_usage_meter';
+
+export interface AccessKeyDataTableRow {
+  nameAndStatus: string;
+  dataUsageAndLimit: string;
+  asCount: string;
+  controlsForId: string;
+}
+
+@customElement('access-key-data-table')
+export class AccessKeyDataTable extends LitElement {
+  @property({type: Array}) accessKeys: AccessKeyDataTableRow[];
+  @property({type: Object}) localize: (messageId: string) => string;
+  @property({type: String}) language: string;
+  @property({type: String}) sortColumn: string;
+  @property({type: Boolean}) sortDescending: boolean;
```

Comment:
Use a template literal to improve readability:

```suggestion
  @property({type: Boolean}) sortDirection: 'asc' | 'desc' = 'asc'`;
```

Not sure if we need a third option there for `''` though. Maybe?